### ISSUE TITLE
Event website: AI in EU Virtual Workshop

### ISSUE THREAD

**Issue by chrisguindon** [EclipseFdn/events.eclipse.org#13]
We need to create a new event website for AI in EU Virtual Workshop

The characteristics of the event will be:
* Date: June 11 (to be confirmed)
* Time: 14:00-17:00 (CEST)

I am thinking that publishing a page for the event on May 18th would be awesome.

We hope to open registration (likely on Eventbrite) on that week of May. 

We will have sponsors.

**Comment by chrisguindon** [comment]
Gael, 

Before we get started, we need some text:

**1. Jumbotron:**
Title: AI in EU Virtual Workshop
Slogan (1 line description):

**2. About the event:** 
(2-3 paragraph about the event)

**Comment by chrisguindon** [comment]
Anne/Gael,

This is an example of an event website:
https://iot.eclipse.org/eclipse-virtual-iot-2020/

This might help you visualize where the text will go!

**Comment by gaelblondelle** [comment]
Hi Chris, 
Date for the event is June 11th, not June 1st.

Still thinking about the exact Title, Slogan and description of the event. 
Will provide that early tomorrow.

Cheers,
Gaël

**Comment by chrisguindon** [comment]
Thanks @gaelblondelle 

I made the edit to my comment.

**Comment by chrisguindon** [comment]
@annejacko @gaelblondelle We need to define a url for this new website.

I am currently going with events.eclipse.org/2020/ai-eu-virtual-workshop/ but feel free to suggest something better!

The current work in progress is available here:
https://deploy-preview-16--eclipsefdn-events.netlify.app/2020/ai-eu-virtual-workshop/

**Comment by annejacko** [comment]
@chrisguindon @gaelblondelle  - Chris,  I just checked with Gael and the official name of the event is Open Source AI Workshop. So let's use this URL instead:
events.eclipse.org/2020/open-source-ai-workshop

Gael is working on the other items you asked for (Jumbotron, description).

**Comment by chrisguindon** [comment]
@annejacko @gaelblondelle I made the requested change:
https://deploy-preview-16--eclipsefdn-events.netlify.app/2020/open-source-ai-workshop/

We are starting the build the page not. Please ignore all the layout problems as we haven't started implementing the design that @ChristieWittEclipse created for #12

**Comment by chrisguindon** [comment]
@annejacko If you are interested, these are the files with the content for building the webpage:
https://github.com/EclipseFdn/events.eclipse.org/pull/16/files

**Comment by annejacko** [comment]
Thanks, Chris. Gael and I will take a look at it on Monday.

Anne


Anne Jacko
Conference Queen | Eclipse Foundation, Inc.
Community. Code. Collaboration. 
Join us at our virtual event:  EclipseCon 2020 <https://www.eclipsecon.org/2020> October 20-22

**Comment by annejacko** [comment]
@chrisguindon Gael and I looked at the website -- looks good so far. We will not need the second CTA button that currently says "Submit a Talk" -- likely the only CTA will be "Register Now."

**Comment by chrisguindon** [comment]
@annejacko No problem!

Most of the content you see are placeholders!

I will adjust once I see the content submitted on this issue!

**Comment by annejacko** [comment]
@chrisguindon Here is the event description for the website:
The Open Source AI workshop brings together European open source and AI experts, as well as top AI open source projects. This is the first virtual event organized by the Eclipse Foundation about Open Source AI, with the goal of gathering a larger community around AI in the Eclipse ecosystem.

**Comment by chrisguindon** [comment]
@annejacko I made the requested content change:
https://deploy-preview-16--eclipsefdn-events.netlify.app/2020/open-source-ai-workshop/

Let me know whenever you wish to publish this event website. 

We can remove the sections where we don't have content such as the agenda and add them back later once the content is ready.

**Comment by annejacko** [comment]
@chrisguindon We just updated the time of the event. The new time is 9:00 - 12:00 CEST. 

I think we can wait a few days before publishing, and by then I may have more content.

**Comment by chrisguindon** [comment]
@annejacko I made the update to the site:
https://deploy-preview-16--eclipsefdn-events.netlify.app/2020/open-source-ai-workshop/

@ChristieWittEclipse is now included in the preview.

**Comment by annejacko** [comment]
@chrisguindon @ChristieWittEclipse The page is looking good! I like the background on the speaker section.

**Comment by annejacko** [comment]
@chrisguindon @gaelblondelle We are ready to publish the website. Let me know if I missed anything we need at this point. Note that one session in the agenda is still TBD so I assume we should leave it off the agenda for now (or delay adding the agenda until it's confirmed).

Front page content
- We have so slogan, so let's leave off that line of text at the top
- Registration: https://open-source-ai-workshop.eventbrite.com
- Speakers: Attaching a zip folder with the speaker photos. Their exact names, job titles, and companies are in the Agenda worksheet.
[OS AI speaker photos.zip](https://github.com/EclipseFdn/events.eclipse.org/files/4659339/OS.AI.speaker.photos.zip)
- Agenda: Note that I added a third track called "Panel" - Worksheet with data for agenda is here: 
https://docs.google.com/spreadsheets/d/19OjqetDhU37ErNNgqhBVmak_8m_0jMyV/edit?ts=5ec58bc0#gid=1263288336
- No display of Program Committee
- Sponsors: Attaching zip folder with two sponsor logos. Please list Huawei first since they are a higher level.
[OS AI sponsor logos.zip](https://github.com/EclipseFdn/events.eclipse.org/files/4659338/OS.AI.sponsor.logos.zip)

**Comment by annejacko** [comment]
@chrisguindon I have an update about the agenda. (I tried to update the worksheet but I think you are editing it offline and I couldn't save the changes.)  For the session where the speakers are not yet confirmed at 9:45, we can list the session, but not list the speaker names yet.

**Comment by chrisguindon** [comment]
@annejacko,

The content is now on the staging site:
https://deploy-preview-16--eclipsefdn-events.netlify.app/2020/ai-eu-virtual-workshop/

Let us know what you think if you give us your +1 we can push this live!

Cheers,

**Comment by annejacko** [comment]
@chrisguindon Thx for the updates. Looking good! A couple of things with the Agenda:
- "Panel on open..."  The time should be 11:10 - 11:30, and the Track should be Panel
- "Online poll..." The Track and speaker name should be empty

@gaelblondelle Can you please review?

**Comment by annejacko** [comment]
@chrisguindon @gaelblondelle Hey Chris -- Gael has reviewed, and just a couple of things -- and then we can go live. Re the sponsor layout -- I noticed that too... we don't plan to have more sponsors, so having them centered would be better!

 Gael's comments:
- Agenda: remove the "online poll line". That's not formally part of the schedule.
- Sponsors: would look better if we have the two logo centered on the page.

**Comment by chrisguindon** [comment]
@annejacko I made the change to the preview:
https://deploy-preview-16--eclipsefdn-events.netlify.app/2020/ai-eu-virtual-workshop/

If all is well, please let us know and we will make this available on the production site.

**Comment by gaelblondelle** [comment]
Hi Chris,
One small thing: "Moderated by Gaël Blondelle" appears twice on the las line of the agenda.
Can you fix it?

Thanks!

**Comment by chrisguindon** [comment]
> Hi Chris,
> One small thing: "Moderated by Gaël Blondelle" appears twice on the las line of the agenda.
> Can you fix it?
> 
> Thanks!

I made the change, the preview site should be update in a minute or so!

**Comment by gaelblondelle** [comment]
Excellent!
Can you publish it?
Also, adding it to events.eclipse.org would be wonderful. 
I have a meeting tomorrow with some specialists of AI in Europe, and I want to show them the website.

Thanks!
Gaël

**Comment by chrisguindon** [comment]
I committed the PR! The site will appear under this URL in the next 10 minutes:
https://events.eclipse.org/2020/open-source-ai-workshop/

@gaelblondelle Someone will need to submit the event to the Eclipse Newsroom for it to appear on events.eclipse.org: https://newsroom.eclipse.org/node/add/events

This is an important step as this allows you to share this event with events.eclipse.org and various other WG site!

**Comment by annejacko** [comment]
I’ll submit it via the Newsroom tomorrow (Tuesday).

Anne


Anne Jacko
Conference Queen | Eclipse Foundation, Inc.
Community. Code. Collaboration. 
Join us at our virtual event:  EclipseCon 2020 <https://www.eclipsecon.org/2020> October 20-22

**Comment by annejacko** [comment]
Correction — I saw that Gael was eager to have it up on the Events page, so I submitted it now instead of waiting until Tuesday my time.

Gael, it goes into moderation, so Shanda will need to approve it before it’s live.


Anne Jacko
Conference Queen | Eclipse Foundation, Inc.
Community. Code. Collaboration. 
Join us at our virtual event:  EclipseCon 2020 <https://www.eclipsecon.org/2020> October 20-22
